### PR TITLE
Added ripe registry app to header

### PR DESCRIPTION
**Pull request by NFSS10**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-registry-ui/pull/1#issuecomment-679190618 |
| Dependencies | -- |
| Decisions | Added ripe-registry to header apps |
| Animated GIF | header_registry_app |

### CODE REVIEW

File: js/apps.js [hunk 1]
```diff
@@ -80,6 +98,12 @@ export const getApps = function(ctx, environment) {
                     link: "https://ripe-white-admin-ui-ci.platforme.com"
                 };
             }
+            if (ctx.hasPermission("ripe-registry.ui")) {
+                apps.whiteAdmin = {
```
Comment (gcandal) [inline js/apps.js:102]: apps.registry

File: js/apps.js [hunk 2]
```diff
@@ -100,6 +124,12 @@ export const getApps = function(ctx, environment) {
                     link: "https://ripe-white-admin-ui-now.platforme.com"
                 };
             }
+            if (ctx.hasPermission("ripe-registry.ui")) {
+                apps.whiteAdmin = {
```
Comment (gcandal) [inline js/apps.js:128]: apps.registry

File: js/apps.js [hunk 3]
```diff
@@ -132,6 +162,12 @@ export const getApps = function(ctx, environment) {
                     link: "https://ripe-orchestra-ui-test.platforme.com"
                 };
             }
+            if (ctx.hasPermission("ripe-registry.ui")) {
+                apps.orchestra = {
```
Comment (gcandal) [inline js/apps.js:166]: apps.registry

File: vue/components/ui/organisms/header/assets/apps/registry.svg [hunk 4]
```diff
@@ -0,0 +1 @@
...8CF9" offset="100%"/></linearGradient></defs><g fill="none"><path fill="url(#a)" opacity=".47" d="M110.508 117.108l-.02-.02.02.02z"/><path d="M75.508 113.852V85.865a5.538 5.538 0 015.538-5.447h27.877a5.538 5.538 0 015.539 5.447v27.987a5.538 5.538 0 01-5.539 5.428h-27.97a5.538 5.538 0 01-5.445-5.428zm38.954-51.083V34.763a5.538 5.538 0 00-5.539-5.428h-27.97a5.538 5.538 0 00-5.538 5.428V62.77a5.538 5.538 0 005.539 5.446h27.97a5.538 5.538 0 005.538-5.446zm-50.437 50.345V85.126a5.538 5.538 0 00-5.447-5.538H30.591a5.538 5.538 0 00-5.428 5.538v27.988a5.428 5.428 0 005.428 5.428h27.987a5.538 5.538 0 005.447-5.428zm-48-76.394L4.32 44.308a5.428 5.428 0 00-1.588 7.569l7.625 11.76a5.538 5.538 0 007.532 1.588L29.631 57.6a5.538 5.538 0 001.606-7.532l-7.643-11.742a5.538 5.538 0 00-7.514-1.606h-.055zm40.08 20.862l9.23-10.524a5.538 5.538 0 00-.48-7.68l-10.504-9.23a5.538 5.538 0 00-7.68.48L37.44 41.132a5.538 5.538 0 00.48 7.68l10.505 9.231a5.538 5.538 0 007.735-.461h-.055zM14.492 15.637l7.385 11.926a5.538 5.538 0 007.495 1.846l11.926-7.384a5.538 5.538 0 001.847-7.496L35.76 2.603A5.538 5.538 0 0028.265.757L16.338 8.142a5.428 5.428 0 00-1.846 7.495z" transform="translate(3)" fill="url(#b)"/></g></svg>
```
Comment (gcandal) [inline vue/components/ui/organisms/header/assets/apps/registry.svg:1]: `npm run pretty`

**Review by gcandal — CHANGES_REQUESTED** [review]

File: vue/components/ui/organisms/header/assets/apps/registry.svg (hunk 4 above)
Comment (NFSS10) [reply vue/components/ui/organisms/header/assets/apps/registry.svg:1]: I had ran `pretty`, it sets the svg like that

**Review by gcandal — APPROVED** [review]

**Review by joamag — APPROVED** [review]

**Pull request merged**